.github/workflows/krb5.yml

```diff
@@ -11,11 +11,14 @@ jobs:
     # This should be a safe limit for the tests to run.
     timeout-minutes: 5
     steps:
+      - name: workaround asan autoconf issue
+        run: sudo apt-get install gcc-11
+
       - name: Build wolfSSL
         uses: wolfSSL/actions-build-autotools-project@v1
         with:
           path: wolfssl
-          configure: --enable-krb CFLAGS='-fsanitize=address'
+          configure: --enable-krb CC='gcc-11' CFLAGS='-fsanitize=address'
```

Comment:
I prefer `CC="gcc-11 -fsanitize=address"` or `CC="gcc-11" CFLAGS="-fsanitize=address" LDFLAGS="-fsanitize=address"`. I saw an issue with this because linker LDFLAGS did not have sanitizer also set.